### PR TITLE
Update RuntimeClass documentation for beta

### PR DESCRIPTION
**Pull request by tallclair**
Update RuntimeClass documentation for beta.

/milestone 1.14

### CODE REVIEW

**Comment by jimangel** [comment]
@tallclair it appears this doc is ready to review, can you confirm? Also, can I get a technical lgtm?

File: content/en/docs/concepts/containers/runtime-class.md [hunk 1]
```diff
@@ -20,72 +20,61 @@ This page describes the RuntimeClass resource and runtime selection mechanism.
...
-Kubernetes git repo: [kubernetes/cluster/addons/runtimeclass/runtimeclass_crd.yaml][runtimeclass_crd]
-
-Install the CRD with `kubectl apply -f runtimeclass_crd.yaml`.
+### Set Up
 
-[CustomResourceDefinition]: /docs/tasks/access-kubernetes-api/custom-resources/custom-resource-definitions/
-[runtimeclass_crd]: https://github.com/kubernetes/kubernetes/tree/master/cluster/addons/runtimeclass/runtimeclass_crd.yaml
+Prerequisite: Ensure the RuntimeClass feature gate is enabled (it is by default). See [Feature
+Gates](/docs/reference/command-line-tools-reference/feature-gates/) for an explanation of enabling
+feature gates. The `RuntimeClass` feature gate must be enabled on apiservers _and_ kubelets.
```
Comment (kbhawkey) [inline content/en/docs/concepts/containers/runtime-class.md]: nit: update the RuntimeClass feature gate to `beta`:
https://kubernetes.io/docs/reference/command-line-tools-reference/feature-gates/

File: content/en/docs/concepts/containers/runtime-class.md [hunk 2]
```diff
@@ -20,72 +20,61 @@ This page describes the RuntimeClass resource and runtime selection mechanism.
...
-
-As an early alpha feature, there are some additional setup steps that must be taken in order to use
-the RuntimeClass feature:
-
-1. Enable the RuntimeClass feature gate (on apiservers & kubelets, requires version 1.12+)
-2. Install the RuntimeClass CRD
-3. Configure the CRI implementation on nodes (runtime dependent)
-4. Create the corresponding RuntimeClass resources
+{{< warning >}} RuntimeClass includes *breaking* changes in the beta upgrade in v1.14. If you were
+using RuntimeClass prior to v1.14, the following actions are required:
```
Comment (jaredbhatti) [inline content/en/docs/concepts/containers/runtime-class.md]: I think you should make this a separate section with a new header "Upgrading RuntimeClass from Alpha to Beta"

Comment (tallclair) [reply content/en/docs/concepts/containers/runtime-class.md]: Done

File: content/en/docs/concepts/containers/runtime-class.md (hunk 1 above)
Comment (tallclair) [reply content/en/docs/concepts/containers/runtime-class.md]: Done

File: content/en/docs/concepts/containers/runtime-class.md [hunk 3]
```diff
@@ -139,4 +134,31 @@ table](https://github.com/kubernetes-sigs/cri-o/blob/master/docs/crio.conf.5.md#
...
+  meaning it can no longer container `.` characters (in all versions). Valid handlers match the
+  following regular expression: `^[a-z0-9]([-a-z0-9]*[a-z0-9])?$`.
+
+If you were using RuntimeClass Alpha (prior to v1.14), the following actions are required
+
+- RuntimeClass resources must be recreated *after* upgrading to v1.14, and the
+  `runtimeclasses.node.k8s.io` CRD should be manually deleted:
+
+      kubectl delete customresourcedefinitions.apiextensions.k8s.io runtimeclasses.node.k8s.io
+
```
Comment (kbhawkey) [inline content/en/docs/concepts/containers/runtime-class.md]: Thanks for the updates.
nit: The kubectl delete instructions could use a wrapper:
```shell
kubectl delete customresourcedefinitions.apiextensions.k8s.io runtimeclasses.node.k8s.io
```

File: content/en/docs/concepts/containers/runtime-class.md [hunk 4]
```diff
@@ -139,4 +134,31 @@ table](https://github.com/kubernetes-sigs/cri-o/blob/master/docs/crio.conf.5.md#
 See cri-o's config documentation for more details:
 https://github.com/kubernetes-sigs/cri-o/blob/master/cmd/crio/config.go
 
+
```
Comment (kbhawkey) [inline content/en/docs/concepts/containers/runtime-class.md]: nit: above in line 109, under `dockershim`:
Kubernete's built-in dockershim CRI does not support runtime handlers.
Change to Kubernetes

**Comment by tallclair** [comment]
Feedback addressed. Also moved the warning up to the overview, and made the action required louder.

File: content/en/docs/concepts/containers/runtime-class.md [hunk 5]
```diff
@@ -150,7 +150,7 @@ RuntimeClass Beta (v1.14) included the following changes:
   meaning it can no longer container `.` characters (in all versions). Valid handlers match the
   following regular expression: `^[a-z0-9]([-a-z0-9]*[a-z0-9])?$`.
 
-If you were using RuntimeClass Alpha (prior to v1.14), the following **actions are required**:
+**Action Required:** If you were using RuntimeClass Alpha (prior to v1.14), the following actions are required upon upgrade to v1.14:
```
Comment (kbhawkey) [inline content/en/docs/concepts/containers/runtime-class.md]: I re-read the page again and found a few nits as well as suggested changes for readability. Totally optional.
nit: The following actions are required to upgrade from the alpha version of the RuntimeClass feature to the beta version:

File: content/en/docs/concepts/containers/runtime-class.md [hunk 6]
```diff
@@ -150,13 +150,13 @@ RuntimeClass Beta (v1.14) included the following changes:
   meaning it can no longer container `.` characters (in all versions). Valid handlers match the
   following regular expression: `^[a-z0-9]([-a-z0-9]*[a-z0-9])?$`.
 
-If you were using RuntimeClass Alpha (prior to v1.14), the following actions are required
+If you were using RuntimeClass Alpha (prior to v1.14), the following **actions are required**:
```
Comment (kbhawkey) [inline content/en/docs/concepts/containers/runtime-class.md]: nit: line 150, s/container/contain ?

File: content/en/docs/concepts/containers/runtime-class.md [hunk 7]
```diff
@@ -23,12 +29,6 @@ This page describes the RuntimeClass resource and runtime selection mechanism.
 RuntimeClass is a beta feature (as of v1.14) for selecting the container runtime configuration to
 use to run a pod's containers.
```
Comment (kbhawkey) [inline content/en/docs/concepts/containers/runtime-class.md]: nit: I don't think you need to repeat that this feature is beta in 1.14.
possible edit: RuntimeClass is a feature for selecting the container runtime configuration. The container runtime configuration is used to run a Pod's containers.

File: content/en/docs/concepts/containers/runtime-class.md [hunk 8]
```diff
@@ -23,12 +29,6 @@ This page describes the RuntimeClass resource and runtime selection mechanism.
...
 
-{{< warning >}}
-RuntimeClass includes *breaking* changes in the beta upgrade in v1.14. If you were using
-RuntimeClass prior to v1.14, see [Upgrading RuntimeClass from Alpha to
-Beta](#upgrading-runtimeclass-from-alpha-to-beta).
-{{< /warning >}}
-
 ### Set Up
 
 Prerequisite: Ensure the RuntimeClass feature gate is enabled (it is by default). See [Feature
```
Comment (kbhawkey) [inline content/en/docs/concepts/containers/runtime-class.md]: nit: not sure if you need `Prerequisite:`

Comment (kbhawkey) [reply content/en/docs/concepts/containers/runtime-class.md]: nit: line 44 the link labeled `below` to (#cri-documentaiton) needs to be changed to (#cri-documentation). After the change, does the link work?
nit: See the corresponding documentation ... to learn how to configure your CRI (Container Runtime Interface) implementation.

File: content/en/docs/concepts/containers/runtime-class.md (hunk 6 above)
Comment (kbhawkey) [reply content/en/docs/concepts/containers/runtime-class.md]: nit: line 140, possible edit:
The RuntimeClass Beta feature includes the following changes:

line 143: s/CustomResourceDefintion/CustomResourceDefinition

File: content/en/docs/concepts/containers/runtime-class.md (hunk 8 above)
Comment (tallclair) [reply content/en/docs/concepts/containers/runtime-class.md]: Done, but moved acronym expansion to the previous sentence.

**Comment by tallclair** [comment]
Thanks, addressed feedback.

File: content/en/docs/concepts/containers/runtime-class.md [hunk 9]
```diff
@@ -40,8 +40,9 @@ feature gates. The `RuntimeClass` feature gate must be enabled on apiservers _an
 
 #### 1. Configure the CRI implementation on nodes
 
-The configurations available through RuntimeClass are CRI implementation dependent. See the
-corresponding documentation ([below](#cri-documentaiton)) for your CRI implementation for how to configure.
+The configurations available through RuntimeClass are Container Runtime Interface (CRI)
+implementation dependent. See the corresponding documentation ([below](#cri-documentation)) for your
+CRI implementation for how to configure.
```
Comment (kbhawkey) [inline content/en/docs/concepts/containers/runtime-class.md]: Looks good, 👍 
nit: the `below` link should reference (#cri-configuration) on the page, not #cri-documentation?

**Pull request merged**